lua/neotest-jest/init.lua

```diff
@@ -154,30 +154,72 @@ end
 function adapter.discover_positions(path)
   local query = [[
     ; -- Namespaces --
+    ; Matches: `describe(`context`, () => {})`
+    ((call_expression
+      function: (identifier) @func_name (#eq? @func_name "describe")
+      arguments: (arguments (template_string (string_fragment) @namespace.name) (arrow_function))
+    )) @namespace.definition
+
```

Comment:
Is it not possible to combine this query with the existing one and the same for the others? Something like the following works with the query editor in `:InspectTree` for `spec/basic.test.ts` when changing one of the `describe`s to use backticks.

```tsq
((call_expression
    function: (identifier) @func_name (#eq? @func_name "describe")
    arguments: (arguments ([
            (string (string_fragment) @namespace.name)
            (template_string (string_fragment) @namespace.name)
        ]
    ) (arrow_function))
)) @namespace.definition
```